capstone/src/main/webapp/index.html

```diff
@@ -1,18 +1,63 @@
+    <link rel="icon" type="image/png" href="assets/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="assets/favicon-16x16.png" sizes="16x16">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <style>
+      .bd-placeholder-img {
+        font-size: 1.125rem;
+        text-anchor: middle;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+      }
+      @media (min-width: 768px) {
+        .bd-placeholder-img-lg {
+          font-size: 3.5rem;
+        }
+      }
+    </style>
+    <link href="cover.css" rel="stylesheet">
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body class="text-center">
+    <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
+  <header class="masthead mb-auto">
+    <div class="inner">
+      <div class="text-center">
+        <img src="assets/perspective_horizontal_color.png" class="rounded">
+      </div>
+      <h3 class="masthead-brand">Perspective Hub</h3>
+      <nav class="nav nav-masthead justify-content-center">
+        <a class="nav-link active" href="#">Home</a>
+        <a class="nav-link" href="https://support.perspectiveapi.com/s/" target="_blank">Support</a>
+        <a class="nav-link" href="https://medium.com/the-false-positive" target="_blank">Blog</a>
+      </nav>
     </div>
-  </body>
+  </header>
+  <main role="main" class="inner cover">
+    <h1 class="cover-heading">Welcome to Perspective Hub</h1>
+    <p class="lead">Perspective Hub Is An All In One Utillity For Testing Perspective API</p>
+    <p class="lead">
+      <a href="http://perspectiveapi.com/#/home" class="btn btn-lg btn-secondary">Learn more</a>
+    </p>
+  </main>
+  <footer class="mastfoot mt-auto">
+    <div class="inner">
+      <p>Built by Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu.</p>
+    </div>
+  </footer>
+    </div>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+</body>
```

Comment:
Nit: lines 33-57 should be indented two more times, and lines 59-62 should be indented one more time